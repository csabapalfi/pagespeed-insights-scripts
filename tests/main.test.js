const main = require('../lib/main');

describe('main', () => {
  const options = {
    runs: 1,
    url: 'https://www.google.com',
    lighthouse: {enabled: false},
    output: {saveAssets: false},
  };

  it('returns runner', () => {
    expect(main(options)).toMatchSnapshot();
  });
});