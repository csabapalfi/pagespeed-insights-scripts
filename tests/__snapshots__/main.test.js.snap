// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main returns runner 1`] = `
Runner {
  "counter": Counter {
    "maxRuns": 1,
    "run": 0,
  },
  "fetcher": Fetcher {
    "getLighthouseResult": [Function],
    "mapResult": [Function],
  },
  "samples": Array [],
}
`;
